### ISSUE TITLE
Debugging not possible with FireFox Portable

### ISSUE THREAD

**Issue by mws-k** [firefox-devtools/vscode-firefox-debug#256]
The debugger works very well with FireFox installed.Unfortunately, I can't get it to work with the portable version of FireFox (which we use internally). I always get the error message: connect ECONNREFUSED 127.0.0.1:6000".

The entry "devtools-debugger.remote-port" is set to 6000.

How can I bring the two tools together?

**Comment by hbenl** [comment]
Have you tried setting `firefoxExecutable` in your launch config to point to the executable of Firefox portable?

**Comment by mws-k** [comment]
yes of course

**Comment by hbenl** [comment]
The `FirefoxPortable` folder contains an executable `FirefoxPortable.exe`, I guess that you pointed the `firefoxExecutable` launch config setting to that file (at least that's what I tried first), but it turns out that file is just a wrapper for the real Firefox executable in the `App` folder and you need to point your config to that file. This is what worked for me: `"firefoxExecutable": "C:\\Users\\Holger\\Downloads\\FirefoxPortable\\App\\Firefox64\\firefox.exe"`

**Comment by mws-k** [comment]
Yes, I know that too and had set it accordingly. You can see that well in the version of the FireFox started.
